spark/src/test/scala/org/apache/spark/sql/delta/MergeIntoSQLSuite.scala

```diff
@@ -386,8 +386,10 @@ class MergeIntoSQLSuite extends MergeIntoSuiteBase
 
 trait MergeIntoSQLColumnMappingSuiteBase extends DeltaColumnMappingSelectedTestMixin {
   override protected def runOnlyTests: Seq[String] =
-    Seq("schema evolution - new nested column with update non-* and insert * - " +
-      "array of struct - longer target")
+    Seq(
+      "schema evolution - new nested column with update non-* and insert * - " +
+        "array of struct - longer target - on via DeltaSQLConf"
+    )
 }
```

Comment:
I saw a test `SET * with schema evolution`, should we add a test here with the "WITH SCHEMA EVOLUTION" syntax while toggling the autoMerge settings on/off?